CONTRIBUTING.rst

```diff
@@ -41,6 +41,9 @@ Talk to Us
 
 Have questions or need help?
 
+# FLOC-2647 Maybe add a separate section which describes an ideal issue report.
+# And link to the log export documentation in debugging.rst?
```

Comment:
I think a link here would be useful. I had trouble getting to the Debugging page (`using/administering/debugging.html`) from the page into which this file gets integrated (`gettinginvolved/contributing`).

:question: File an issue for @davidoliverSP2 to review the content and relative position of these two pages.